payments-core/src/main/java/com/stripe/android/payments/bankaccount/CollectBankAccountLauncher.kt

```diff
@@ -18,13 +18,15 @@ interface CollectBankAccountLauncher {
     fun presentWithPaymentIntent(
         publishableKey: String,
         clientSecret: String,
-        configuration: CollectBankAccountConfiguration
+        configuration: CollectBankAccountConfiguration,
+        attachToIntent: Boolean = true
```

Comment:
Do we want to expose this to users already using the launcher? If not I'd rather create an internal `CollectBankAccountForPaymentSheetLauncher` that passes this boolean down to the activity.

Reply:
![image](https://user-images.githubusercontent.com/99293320/163282372-4cdf16cd-aeb6-4397-b147-ebacea8f1d35.png)